src/main/java/br/com/alura/comex/processadores/ProcessadorDeCsv.java

```diff
@@ -0,0 +1,61 @@
+package br.com.alura.comex.processadores;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import br.com.alura.comex.pedido.Pedido;
+
+public class ProcessadorDeCsv {
+
+	private ArrayList<Pedido> pedidos = new ArrayList<Pedido>();
+
+	public ArrayList<Pedido> registrarPedidos() {
+		try {
```

Comment:
Prefira interfaces a implementações, quando possível!

```suggestion
	private List<Pedido> pedidos = new ArrayList<Pedido>();

	public List<Pedido> registrarPedidos() {
		try {
```